rviz_common/include/rviz_common/ros_topic_display.hpp

```diff
@@ -183,12 +183,15 @@ class RosTopicDisplay : public _RosTopicDisplay
     }
 
     try {
+      // TODO(wjwwood): update this class to use rclcpp::QoS.
+      auto qos = rclcpp::QoS(rclcpp::QoSInitialization::from_rmw(qos_profile));
```

Comment:
`rclcpp::QoS qos(rclcpp::QoSInitialization::from_rmw(qos_profile));`

Reply:
I'm going to pass on these for now, as it doesn't really change anything I don't want to introduce a stupid typo or something and cause another CI round.